### ISSUE TITLE
Unable to set multiple ports in fortios_firewall_sslsshprofile https block

### ISSUE THREAD

**Issue by freakinhippie** [fortinetdev/terraform-provider-fortios#148]
```
Firmware: FortiOS v6.4.5 build1828 (GA)
Provider: fortinet/fortios/1.10.4
Terraform: 0.14.7
```

I'm trying to create an SSL/SSH profile using the `fortios_firewall_sslsshprofile` resource, but it seems that the `ports` attribute of the `https`/`ftps`/`imaps`/`pop3s`/`smtps` blocks only accept a single number. However, this is a multi-valued field in the CLI.

```hcl
resource "fortios_firewall_sslsshprofile" "testing" {
  name = "testing"

  https {
    ports = "443 8443" # doesn't work
    # ports = "443,8443" # also doesn't work
    # ports = [443, 8443] # also doesn't work
  }
}
```

The above results in this error:

```
Error: Incorrect attribute value type

  on ssl_ssh_profiles.tf line 5, in resource "fortios_firewall_sslsshprofile" "testing":
  5:     ports = "443 8443"

Inappropriate value for attribute "ports": a number is required.
```

Perhaps the input should be a string rather than a number, so that multiple ports can be provided.

Thanks.

**Comment by frankshen01** [comment]
Hi @freakinhippie, thanks for the issue. You are absolutely right, this is a bug, it is caused by API json interface. I will add a patch to the terraform provider to solve this problem. The bug will be fixed and included in the next release (within a week). In the meantime, the following workarounds can also help resolve this issue:

```hcl
resource "fortios_firewall_sslsshprofile" "t1" {
  name = "zzzzz1"

  ssl {
    inspect_all = "disable"
  }

  https {
    ports = 443
  }
  ftps {
    ports = 990
  }
  imaps {
    ports = 993
  }
  pop3s {
    ports = 995
  }
  smtps {
    ports = 465
  }
}

resource "fortios_system_autoscript" "trname1" {
  interval    = 1
  name        = join("_", [fortios_firewall_sslsshprofile.t1.name, "script"])
  output_size = 10
  repeat      = 1
  script      = <<EOF
config firewall ssl-ssh-profile
  edit ${fortios_firewall_sslsshprofile.t1.name}
    config https
      set ports 533 544 555
    end
  end
end
EOF
  start       = "auto"
}
```

validation:
```
labfirewall # config firewall ssl-ssh-profile

labfirewall (ssl-ssh-profile) # edit zzzzz1

labfirewall (zzzzz1) # show
config firewall ssl-ssh-profile
    edit "zzzzz1"
        config https
            set ports 533 544 555
            set status deep-inspection
        end
        config ftps
            set ports 990
            set status deep-inspection
        end
        config imaps
            set ports 993
            set status deep-inspection
        end
        config pop3s
            set ports 995
            set status deep-inspection
        end
        config smtps
            set ports 465
            set status deep-inspection
        end
        config ssh
            set ports 22
            set status disable
        end
    next
end

```
Thank you!

**Comment by frankshen01** [comment]
@freakinhippie  It has been fixed and released in version 1.11.0 of the Terraform FortiOS provider. Please see: https://registry.terraform.io/providers/fortinetdev/fortios/latest and https://registry.terraform.io/providers/fortinetdev/fortios/latest/docs/resources/fortios_firewall_sslsshprofile. Thanks!
